### PR TITLE
Patch CVE-2020-8037 in tcpdump

### PR DESCRIPTION
**Pull request by thcrain-msft**
###### Merge Checklist  

**All** boxes should be checked before merging the PR *(just tick any boxes which don't apply to this PR)*
- [x] The toolchain has been rebuilt successfully (or no changes were made to it)
- [x] The toolchain/worker package manifests are up-to-date
- [x] Any updated packages successfully build (or no packages were changed)
- [x] All package sources are available
- [x] cgmanifest files are up-to-date and sorted (`./cgmanifest.json`, `./toolkit/tools/cgmanifest.json`, `./toolkit/scripts/toolchain/cgmanifest.json`)
- [x] LICENSE-MAP files are up-to-date (`./SPECS/LICENSES-AND-NOTICES/LICENSES-MAP.md`, `./SPECS/LICENSES-AND-NOTICES/LICENSE-EXCEPTIONS.PHOTON`)
- [x] All source files have up-to-date hashes in the `*.signatures.json` files
- [x] `sudo make go-tidy-all` and `sudo make go-test-coverage` pass
- [x] Documentation has been updated to match any changes to the build system
- [x] Ready to merge

---

###### Summary 

Patch CVE-2020-8037 in tcpdump

###### Change Log  

- Take upstream patch for CVE-2020-8037
- Lint tcpdump spec to Mariner style

###### Does this affect the toolchain?  

NO

###### Links to CVEs  
- https://nvd.nist.gov/vuln/detail/CVE-2020-8037

###### Test Methodology

- Local build + ran check for affected package

### CODE REVIEW

File: SPECS/tcpdump/tcpdump.spec [hunk 1]
```diff
@@ -37,43 +38,57 @@ make %{?_smp_mflags} check
 %{_mandir}/man1/*
 
 %changelog
+* Fri Nov 13 2020 Thomas Crain <thcrain@microsoft.com> - 4.9.3-2
```
Comment (anphel31) [inline SPECS/tcpdump/tcpdump.spec:41]: Should be 4.9.3-3

File: SPECS/tcpdump/tcpdump.spec [hunk 2]
```diff
@@ -17,15 +18,15 @@ It allows the user to display TCP/IP and other packets being
...
+%autosetup -p1
 
 %build
 %configure
 make %{?_smp_mflags}
 
 %install
 make DESTDIR=%{buildroot} install
-find %{buildroot} -name '*.la' -delete
+find %{buildroot} -type f -name "*.la" -delete -print
```
Comment (PawelWMS) [inline SPECS/tcpdump/tcpdump.spec:29]: I've got nothing against it, just curious: why `-print`? Is it something the linter is adding automatically?

**Review by nicogbg — APPROVED** [review]

**Review by PawelWMS — APPROVED** [review]

**Review by christopherco — APPROVED** [review]

**Review by dmcilvaney — APPROVED** [review]

File: SPECS/tcpdump/tcpdump.spec (hunk 2 above)
Comment (thcrain-msft) [reply SPECS/tcpdump/tcpdump.spec:29]: We decided to unify all forms of *.la deletion, and this just happened to be what the openSUSE version of the linter suggested.

File: SPECS/tcpdump/tcpdump.spec (hunk 1 above)
Comment (thcrain-msft) [reply SPECS/tcpdump/tcpdump.spec]: Resolved, thanks!

**Pull request merged**